config/config.properties

```diff
@@ -29,82 +29,82 @@
 # Valid values: Positive integer in the valid TCP port ranges.
-pravegaservice.listeningPort=12345
+pravegaservice.service.listener.port=12345
 
 # Full URL (host:port) where to find a ZooKeeper that can be used for coordinating this Pravega Cluster.
 # Required.
-pravegaservice.zkURL=localhost:2181
+pravegaservice.zk.connect.uri=localhost:2181
 
 # Number of milliseconds to wait between ZooKeeper retries.
 # Valid values: Positive integer.
 # Recommended values: Between 5000 and 10000 (too high will cause failover delays).
-#pravegaservice.zkRetrySleepMs=5000
+#pravegaservice.zk.connect.retries.interval.milliseconds=5000
 
 # Number of times to retry a failed attempt to contact ZooKeeper.
 # Valid values: Positive integer.
 # Recommended values: Between 3 and 5.
-#pravegaservice.zkRetryCount=5
+#pravegaservice.zk.connect.retries.count.max=5
 
 # Session timeout for ZooKeeper.
 # Valid values: Positive integer.
-#pravegaservice.zkSessionTimeoutMs=10000
+#pravegaservice.zk.connect.sessionTimeout.milliseconds=10000
 
 # Enable security for the ZK client.
-#pravegaservice.secureZK=false
+#pravegaservice.zk.connect.security.enable=false
 
 # Location of trust store for secure connection to ZK.
-#pravegaservice.zkTrustStore=
+#pravegaservice.zk.connect.security.tls.trustStore.location=
 
 #Location of password file to access the trust store.
-#pravegaservice.zkTrustStorePasswordPath=
+#pravegaservice.zk.connect.security.tls.trustStore.pwd.location=
 
 # DataLog implementation for Tier 1 storage.
 # Valid values: BOOKKEEPER, INMEMORY.
 # Default value: BOOKKEEPER
-pravegaservice.dataLogImplementation=BOOKKEEPER
+pravegaservice.dataLog.impl.name=BOOKKEEPER
 
 # Storage implementation for Tier 2 storage.
 # Valid values: HDFS, FILESYSTEM, EXTENDEDS3, INMEMORY.
 # Default value: HDFS
-# pravegaservice.storageImplementation=HDFS
+# pravegaservice.storage.impl.name=HDFS
 
 # Storage NO-OP Mode: in No-Op mode, user stream segment writing is no-oped; user stream segment reading is not supported.
 # This mode is used to avoid storage interference in testing while still keep the system functioning as usual.
 # NOTE: pravegaservice.storageImplementation is still used to store metadata and system segments, which are required for the functioning of the Pravega Cluster.
 # WARNING: Do NOT set NO-OP Mode to true in production environment! Data loss will happen!
 # Valid values: true, false
 # Default value: false
-#storageextra.storageNoOpMode=false
+#storageextra.noOpMode.enable=false
 
 # When Storage NO-OP Mode is true, storageWriteNoOpModeLatencyMillis is applied to write operation.
 # It is used to compensate the supposed latency if the storage is in operation, in order to make the test as real as possible.
 # This latency has no effect if storage is not in No-Op mode.
 # Valid values: non-negative integer
 # Default vale: 20 (milliseconds)
-#storageextra.storageWriteNoOpLatencyMillis=20
+#storageextra.write.noOp.latency.milliseconds=20
```

Comment:
`write` here seems to refer to an IO write, should it be `writer` instead? Also, is it related to the previous configuration option above such that we need to add `.noOp.` there too?

Reply:
@andreipaduroiu could you please clarify the purpose of this configuration item? Could you also suggest a better name for this?

Reply:
@kevinhan88 wrote this thing. I would drop the `write` and just emphasize `noOp.

Reply:
Actually, please be consistent with the above property. You use `noOpMode` and here you use `noOp`.

Reply:
Fixed. Also, I left the `write` in the name as it is apparently referring to `write` latency.